docs/docker.md

```diff
@@ -0,0 +1,40 @@
+# Docker
+
+It's possible to run Ollama with Docker or Docker Compose.
+
+The official [Ollama Docker image](https://hub.docker.com/r/ollama/ollama)
+`ollama/ollama` is available on Docker Hub.
+
+## Docker Hub
+
+To interact with the official Ollama images:
+
+```bash
+# CPU only
+docker run -d -v ollama:/root/.ollama -p 11434:11434 --name ollama ollama/ollama
+# With GPU
+docker run -d --gpus=all -v ollama:/root/.ollama -p 11434:11434 --name ollama ollama/ollama
+```
```

Comment:
I think you should separate the comments

### CPU only

```bash
docker run -d -v ollama:/root/.ollama -p 11434:11434 --name ollama ollama/ollama
```

### With GPU


```bash
docker run -d --gpus=all -v ollama:/root/.ollama -p 11434:11434 --name ollama ollama/ollama
```

Reply:
Pushed this change